Documentation/installing.md

```diff
@@ -0,0 +1,185 @@
+# Installing IronPython Released Binaries
+
+IronPython can be used as a standalone interpreter, or embedded within another .NET application. For embedding, the recommended approach is to install IronPython within the target project using NuGet. This document describes various ways of installing the standalone interpreter on the target system. Once installed, IronPython is invocable from the command line as `ipy`. Use `ipy -h` for a list of available commadline options. Use `ipy -VV` to produce version information for reporting issues.
+
+## IronPython on .NET (Core)
+
+Since .NET is a cross-platform framework, the instructions in this section apply to all supported operating systems (Windows, Linux, macOS), unless explicitly indicated.
+
+Currently IronPython is supported on .NET 6.0 and .NET Core 3.1.
```

Comment:
.NET Core 3.1 just went out of support, wondering if we should be mentioning it.

Reply:
OK, I'll remove this line; one place less to keep track of when supported frameworks change.